starfyre/exceptions.py

```diff
@@ -16,4 +16,5 @@ class UnknownTagError(Exception):
    Attributes:
         message (str): A description of the error.
    """
-   pass
\ No newline at end of file
+   def __init__(self, message='Unknown tag found.'):
+    super().__init__(message)
```

Comment:
Can you please explain why we need this?

Reply:
Okay, thanks fo asking!

This line calls the `__init__` method of the parent class, `Exception`. By doing so, the custom exception class inherits the behavior of the built-in `Exception` class and ensures that the exception is properly initialized with the provided error message.

I believe it makes the code a bit easier to understand. We know that it is dependent on the `Exception` class.